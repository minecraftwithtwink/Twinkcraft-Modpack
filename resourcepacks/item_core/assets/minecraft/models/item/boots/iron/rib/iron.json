{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/iron/rib",
        "layer1": "minecraft:trims/items/boots_trim/rib_iron_darker"
    }
}