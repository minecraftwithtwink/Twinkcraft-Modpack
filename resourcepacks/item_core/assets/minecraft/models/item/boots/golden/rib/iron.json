{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/golden/rib",
        "layer1": "minecraft:trims/items/boots_trim/rib_iron"
    }
}